src/frontend/src/components/PersonsCarousel/PersonsCarousel.jsx

```diff
@@ -11,7 +11,7 @@ export default function PersonsCarousel({
       id={person.id}
       pictureUri={person.pictureUri}
       name={person.name}
-      role={person.role}
+      role={Array.isArray(person.role) ? person.role.join(', ') : person.role}
       additionalInfo={person.additionalInfo}
       isLoading={loading}
     />
```

Comment:
The character the person plays should also be shown when its as a cast member on the media page